CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog (weaverbird npm package)
 
+## [0.91.3] - 2022-10-10
+
+### Added
+- Add simplify step
+
+### Fixed
+- Fix convert step when converting to float
+- Fix unpivot by converting all targeted columns to float
+- Fix "notmatches" operator for backends using REGEXP
+- Fix split step quoting
+- Fix "matches" operator for IfThenElse with pandas.NA values
```

Comment:
I think it is okay to list only front-end changes here, back-end changes are listed in `server/CHANGELOG.md`